### DISCUSSION TITLE
Can not start service zigbee2mqtt tcp4 0.0.0.0:8080  address already in use

### DISCUSSION THREAD

**Discussion by mbolanos15** [Koenkk/zigbee2mqtt#17114] [Devices]
Hello
Sorry if this is a stupid question, but I'm new to this...
There is a program that it use the port 8080 (I need it) and Can not start service zigbee2mqtt. Is it possible to use other port?. any recommendation? Thanks you.
Regards.

**Comment by junosuarez** [comment]
How are you running zigbee2mqtt? If you're using docker, you can change the port it's mapped to with the startup command. From the [docs example](https://www.zigbee2mqtt.io/guide/installation/02_docker.html#running-the-container), change the `-p` section (for `port`):
`-p 8888:8080` where `8888` is whatever port you want to use that is available on your machine. 

If you're running zigbee2mqtt some other way, use the [`frontend.port` configuration option](https://www.zigbee2mqtt.io/guide/configuration/frontend.html#advanced-configuration):
```yaml
# configuration.yaml

frontend:
  # Optional, default 8080
  port: 8888
```

**Reply by mbolanos15** [reply]
Thanks Juno, 
I am running zigbee2mqtt using docker-compose.
Would this be a valid configuration?

services:
  zigbee2mqtt:
    container_name: zigbee2mqtt
    image: koenkk/zigbee2mqtt
    restart: unless-stopped
    volumes:
      - /docker/zigbee2mqtt/data:/app/data
      - /run/udev:/run/udev:ro
    ports:
      # Frontend port
      - 8079:8080
    environment:
      - TZ=Europe/Berlin
    devices:
      # Make sure this matched your adapter location
      - /dev/ttyUSB0:/dev/ttyUSB0

Can you recommend a port that does not use any important add-ons?
Thanks you so much for your help.

Regads.

**Comment by junosuarez** [comment]
> Would this be a valid configuration?

At a glance, it seems ok, assuming it's properly indented (whitespace matters in yaml).

> Can you recommend a port that does not use any important add-ons?

The host port doesn't really matter, as long as it doesn't conflict with anything else you have running.
